packages/client/components/JiraScopingSearchCurrentFilters.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import {JiraScopingSearchCurrentFilters_meeting$key} from '../__generated__/JiraScopingSearchCurrentFilters_meeting.graphql'
+
+const Wrapper = styled('div')({
+  width: '100%',
+  display: 'flex',
+  paddingLeft: '72px',
+  paddingTop: '8px'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 500,
+  whiteSpace: 'nowrap'
+})
+
+const Items = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 600,
+  fontStyle: 'italic',
+  padding: '0px 24px 0px 4px',
+  width: '100%',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
+})
+
+interface Props {
+  meetingRef: JiraScopingSearchCurrentFilters_meeting$key
+}
+
+const JiraScopingSearchCurrentFilters = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment JiraScopingSearchCurrentFilters_meeting on PokerMeeting {
+        jiraSearchQuery {
+          projectKeyFilters
+        }
+        viewerMeetingMember {
+          teamMember {
+            integrations {
+              atlassian {
+                projects {
+                  id
+                  name
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    meetingRef
+  )
+  const {jiraSearchQuery} = meeting
+  const {viewerMeetingMember} = meeting
+  const {teamMember} = viewerMeetingMember!
+  const {integrations} = teamMember
+  const {atlassian} = integrations
+  const {projects} = atlassian!
+  const {projectKeyFilters} = jiraSearchQuery
+  const projectFilterNames = projectKeyFilters.map((projectId, idx) => {
+    const project = projects.find(({id}) => id === projectId)!
```

Comment:
This has a worst-case performance of O(n*m). Still, I decided not to memorize it because the component tree structure means that this component usually only re-renders when `projectKeyFilters` changes anyway. Also, in practice, `projectKeyFilters` will be <10 and the total number of projects will normally be <100.